### DISCUSSION TITLE
Reg: Trino

### DISCUSSION THREAD

**Discussion by bazooka720** [DataJunction/dj#1060] [Q&A]
Hello Folks: We chanced across this project which seems very relevant for our needs. Curious if this currently can work with Trino and also upstream do you have functionality to enable connectivity with BI tools like Tableau, PowerBI. 

Trying to understand how this is different from the cube.dev project. Would be great to understand as we are looking to build a semantic layer on top of Trino. We were hoping to use iceberg for MVs as well @agorajek @betodealmeida

**Comment by agorajek** [comment]
Hey @bazooka720 . This project is similar to cube.dev, but it is only 2 years old. Trino compatibility for generating SQL is ready out-of-the-box, but we also support Druid for faster data access. There is currently no native driver for Tableau or JDBC-like connectors, but we are going to add something in the near future. We have tried custom connection to Superset and this vis tool one will probably be the 1st one to have an official support.

If you'd like to join our weekly sync and discuss your case please let me know. Most of the current developers are with me at Netflix and we sure could share some info from our side as well.
